_source/user-guide/infrastructure-monitoring/managing-metrics-account.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: article
+title: Managing your Metrics account
+permalink: /user-guide/infrastructure-monitoring/metrics-account.html
+flags:
+  admin: true
+  logzio-plan:
+tags:
+contributors:
+  - shalper
+---
+
+By default, your Metrics account is found under your main log monitoring account.
+To open it, log into your Logz.io Operations account and select the
+[Metrics tab](https://app.logz.io/#/dashboard/grafana/).
+
+![Navigate to your metrics account](https://dytvr9ot2sszz.cloudfront.net/logz-docs/grafana/reach-metrics.png)
```

Comment:
There's a lot going on in this image - I recommend cropping it in to just the areas you need